h/views/api/groups.py

```diff
@@ -81,6 +83,34 @@ def remove_member(group, request):
     return HTTPNoContent()
 
 
+@api_config(route_name='api.group_member',
+            request_method='POST',
+            link_name='group.member.add',
+            description='Add the current user to a group.',
+            effective_principals=security.Authenticated)
+def add_member(request):
+    """Add a member to a given group."""
+    client = request_auth_client(request)
+
+    user_svc = request.find_service(name='user')
+    group_svc = request.find_service(name='group')
+    groupfinder_svc = request.find_service(IGroupService)
+
+    group = groupfinder_svc.find(request.matchdict['pubid'])
+    user = user_svc.fetch(request.matchdict['user'],
+                          client.authority)
+    if user is None:
+        raise HTTPNotFound()
+    if group is None:
```

Comment:
This _may_ already happen automatically if you use Pyramid traversal, but you'll have to test...